.dev/compare_branches.R

```diff
@@ -304,7 +304,17 @@ if (is_branch) {
 } else {
   message("Comparing PR#", pr, " to master")
 }
-message("Comparing output of lint_dir run for the following packages: ", toString(basename(packages)))
+if (length(packages) > 50L) {
```

Comment:
wdyt about using `glue::glue_collapse()` instead?

```
> glue::glue_collapse(letters, ", ", last = " and ", width = 10)
a, b, c...
> glue::glue_collapse(letters, ", ", last = " and ", width = Inf)
a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, u, v, w, x, y and z
```

Reply:
I think probably overkill for a verbose message in an internal script like this... `toString` also has a `width` argument FWIW